packages/transport-websockets/examples/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please keep the examples in the examples folder, it's much easier for people visiting the repo to find them that way, since they don't have to dig through the monorepo folders.